### ISSUE TITLE
Allow to override `E` elements

### ISSUE THREAD

**Issue by qcoumes** [jurismarches/luqum#73]
I need to customize the behavior of some elements inside `ElasticsearchQueryBuilder`. Sadly it is not currently possible to just change these element without overriding every method to use my custom element.

Could you consider making these elements an attribute of the class so that we can override them more easily, i.e. something like :

```python
class ElasticsearchQueryBuilder(LuceneTreeVisitorV2):
    
    E_MUST = EMust
    E_SHOULD = Eshould
    [...]
```

I can make a PR if you agree with the idea.

**Comment by alexgarel** [comment]
Sounds good to me @qcoumes

**Comment by alexgarel** [comment]
Resolved by @qcoumes PR #75 !
